### PR TITLE
Update runtime to 6.7, ...

### PR DESCRIPTION
**Pull request by sungsphinx**
- Remove `--env=QT_QPA_PLATFORM=xcb`
- Use `fallback-x11` instead of `x11`
- Bump runtime to 6.7
- Update dependencies/modules/whatever :melting_face:

### CODE REVIEW

**Comment by tsdgeos** [comment]
Have you tested this?

I don't how your changes regarding run_kontact can work 

Also i would very much prefer if the QT_QPA_PLATFORM thing was a different commit/PR

**Review by travier — APPROVED** [review]
Thanks, LGTM. Have you tested the build from the bot?

File: org.kde.kontact.json [hunk 1]
```diff
@@ -1230,7 +1229,8 @@
                 }
             ],
             "post-install": [
-                "install run_kontact.sh /app/bin/run_kontact.sh"
+                "mv /app/bin/{kontact,kontact-bin}",
```
Comment (tsdgeos) [inline org.kde.kontact.json:1232]: What's the point of this?

You shot yourself on the feet with it so i have to think that it serves some purpose?

File: org.kde.kontact.json [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
+    "runtime-version": "6.7",
     "base": "io.qt.qtwebengine.BaseApp",
-    "base-version": "6.6",
+    "base-version": "6.7",
     "sdk": "org.kde.Sdk",
     "sdk-extensions": [
         "org.freedesktop.Sdk.Extension.rust-stable"
     ],
-    "command": "run_kontact.sh",
+    "command": "kontact",
```
Comment (tsdgeos) [inline org.kde.kontact.json:12]: I do not like this, the old way makes it super clear that a script that is doing more things than run kontact is happening, this is hidden with a rename very down in the file, what's the benefit of this?

**Comment by tsdgeos** [comment]
DO NOT MERGE THIS

I am waiting for my comments to be addressed.

I'm quite unhappy you just decided to press the "Resolve conversation" without answering me.

**Review by travier — APPROVED** [review]
LGTM. The script rename has been removed.

**Comment by tsdgeos** [comment]
Please next time write something saying you've done such a change

**Pull request merged**